### DISCUSSION TITLE
Mouse wheel handling

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#824] [Q&A] [answered]
Hello,

I am trying to add mouse wheel handling to my VTM project. Is mouse wheel handling already included in the official project?

**Comment by devemux86** [comment]
Mouse wheel already performs map zoom on desktop.
You can test it with [vtm-playground](https://github.com/mapsforge/vtm/tree/master/vtm-playground) samples.

**Reply by demiantres** [reply]
Can you point me to the exact example? I found only the "MapPanelMouseListener" implementation which seems to be tied to the Mapsforge package.

**Comment by devemux86** [comment, marked as answer]
All desktop examples in [vtm-playground](https://github.com/mapsforge/vtm/tree/master/vtm-playground) work similarly.

VTM uses [libGDX](https://libgdx.com/) on desktop (with [InputHandler](https://github.com/mapsforge/vtm/blob/master/vtm-gdx/src/org/oscim/gdx/InputHandler.java) class).

**Reply by demiantres** [reply]
Thanks, works great!
